app/views/submissions/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<div>
+  <h2><%=@submission.title%></h2>
+  <h3><%=@submission.submitted_to%></h3>
+  <h3>Poems in this submission</h3>
+  <ul>
+    <% if @submission&.poems&.each do |poem| %>
+      <li><%=poem.title%></li>
+      <% end %>
```

Comment:
Closes the `if`. The indentation on the line above is off.

How about adding Hound to this repo?